ledger/evalIndexer.go

```diff
@@ -25,30 +25,31 @@ import (
 	"github.com/algorand/go-algorand/data/basics"
```

Comment:
could you also rename `evalIndexer` to `evalindex` or something to be consistent with naming all other files?

Reply:
yes